src/content/datasheets/core-datasheet.md

```diff
@@ -313,22 +313,27 @@ Mechanical drawings of the Core are available [here.](https://github.com/spark/c
 <table border = '1'>
    <tr>
       <th>Parameter</th>
-      <th>Value</th>
+      <th>Value (imperial)</th>
+      <th>Value (metric)</th>
    </tr>
    <tr>
       <td>Length</td>
       <td>1.47"</td>
+      <td>37.27 mm</td>
```

Comment:
Please do make the conversion match what is currently there, I.e. 37.34mm. We can dig into specs to find out what the measurement should be and what the tolerance is (but I would not just blindly trust one caliper reading as the new truth ;-) ) thanks for adding!